src/spring/azext_spring/_client_factory.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 from azure.cli.core.commands.client_factory import get_mgmt_service_client
 from azure.cli.core.profiles import ResourceType
-from .vendored_sdks.appplatform.v2024_01_01_preview import AppPlatformManagementClient
+from .vendored_sdks.appplatform.v2024_05_01_preview import AppPlatformManagementClient
```

Comment:
Backend is not fully released, I think we cannot merge this PR

Reply:
thanks for your reminder, will merge it after code release.